core/sql/executor/ex_hash_grby.cpp

```diff
@@ -1027,8 +1027,10 @@ void ex_hash_grby_tcb::returnResultCurrentRow(HashRow * dataPointer)
   upParentEntry->upState.downIndex = parentQueue_.down->getHeadIndex();
   
   // if stats are to be collected, collect them.
-  if (bmoStats_)
-    bmoStats_->incActualRowsReturned();
+  ExOperStats *statsEntry = getStatsEntry();
```

Comment:
Is the member bmoStats_ no longer used then? Should it be deleted?

Reply:
bmoStats_ is also used for reporting memory-related counters, so we still need it.